src/platform/baytrail/include/platform/drivers/dw-dma.h

```diff
@@ -11,6 +11,14 @@
 #define __PLATFORM_DRIVERS_DW_DMA_H__
 
 #include <sof/bit.h>
+#include <config.h>
+
+/* number of supported DW-DMACs */
+#if CONFIG_CHERRYTRAIL
+#define PLATFORM_NUM_DW_DMACS	3
```

Comment:
Keeping this at 2 for the moment due to issue reported by @cujomalainey

Reply:
use the extra DMA config instead https://github.com/thesofproject/sof/blob/master/src/platform/baytrail/include/platform/lib/dma.h#L16

Reply:
Done.